src/pages/company_directory.js

```diff
@@ -8,6 +8,7 @@ import Nav from "../components/nav"
 import Typography from "@material-ui/core/Typography"
 import TextField from "@material-ui/core/TextField"
 import CompanyCard from "../components/company_card"
+import { Helmet } from "react-helmet"
```

Comment:
Is it possible to add this to another Component that is imported in all pages? Then we would only need it once. Maybe `Header` is a good candidate 🤔

Reply:
I will look into how to do that

Reply:
oh nice, yes header did work. I'll have to pick your brain on that sometime for my own knowledge